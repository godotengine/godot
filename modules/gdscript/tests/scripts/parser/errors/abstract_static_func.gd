extends RefCounted

abstract class A:
	# Currently, an abstract function cannot be static.
	abstract static func f()

func test():
	pass
