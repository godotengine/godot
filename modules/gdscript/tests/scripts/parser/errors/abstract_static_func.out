GDTEST_PARSER_ERROR
Expected "class" or "func" after "abstract".
